src/Rocketeer/Strategies/Deploy/LocalCloneStrategy.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Rocketeer\Strategies\Deploy;
+use Rocketeer\Strategies\Deploy\SyncStrategy;
+use Rocketeer\Bash;
+
+class LocalCloneStrategy extends SyncStrategy
+{
+    /**
+     * @type string
+     */
+    protected $description = 'Uses rsync to create or update a release from local temporary cloned repository';
+    
+    /**
+     * Deploy a new clean copy of the application.
+     *
+     * @param string|null $destination
+     *
+     * @return bool
+     */
+    public function deploy($destination = null)
+    {
+        if (!$destination) {
+            $destination = $this->releasesManager->getCurrentReleasePath();
+        }
+        
+        // Create receiveing folder
+        $this->createFolder($destination);
+        
+        $tmpFolderPath = '.rocketeer/checkout/tmp/' . time() . '/';
```

Comment:
Hm I think the temporary folder should use the storage folder instead of the configuration folder, no?